devices/sam/samv70.xml

```diff
@@ -569,24 +537,18 @@
       </gpio>
       <gpio device-pin="q" port="C" pin="9">
         <signal driver="ebi" name="nandoe" function="a"/>
-        <signal device-flash="19|20" device-variant="a" device-package="a" device-grade="a" driver="sdramc" name="nandoe" function="a"/>
-        <signal device-flash="19|20" device-variant="a" device-package="c" device-grade="b" driver="sdramc" name="nandoe" function="a"/>
         <signal device-flash="19|20" device-variant="a" device-package="a" device-grade="a" driver="smc" name="nandoe" function="a"/>
         <signal device-flash="19|20" device-variant="a" device-package="c" device-grade="b" driver="smc" name="nandoe" function="a"/>
         <signal driver="tc" instance="2" name="tiob" function="b" index="7"/>
       </gpio>
       <gpio device-pin="q" port="C" pin="10">
         <signal driver="ebi" name="nandwe" function="a"/>
-        <signal device-flash="19|20" device-variant="a" device-package="a" device-grade="a" driver="sdramc" name="nandwe" function="a"/>
-        <signal device-flash="19|20" device-variant="a" device-package="c" device-grade="b" driver="sdramc" name="nandwe" function="a"/>
         <signal device-flash="19|20" device-variant="a" device-package="a" device-grade="a" driver="smc" name="nandwe" function="a"/>
         <signal device-flash="19|20" device-variant="a" device-package="c" device-grade="b" driver="smc" name="nandwe" function="a"/>
         <signal driver="tc" instance="2" name="tclk" function="b" index="7"/>
       </gpio>
       <gpio device-pin="q" port="C" pin="11">
         <signal driver="ebi" name="nrd" function="a"/>
-        <signal device-flash="19|20" device-variant="a" device-package="a" device-grade="a" driver="sdramc" name="nrd" function="a"/>
-        <signal device-flash="19|20" device-variant="a" device-package="c" device-grade="b" driver="sdramc" name="nrd" function="a"/>
         <signal device-flash="19|20" device-variant="a" device-package="a" device-grade="a" driver="smc" name="nrd" function="a"/>
         <signal device-flash="19|20" device-variant="a" device-package="c" device-grade="b" driver="smc" name="nrd" function="a"/>
```

Comment:
These constraints look really weird. At first there are no other valid values for `device-flash` other than `19` and `20`. There is a second problem with the `device-package` values. modm assumes the identifier is fixed-length but there are two-letter `device-package` values:
![atsam_v70](https://user-images.githubusercontent.com/25187160/179351704-6b9a1c5b-af89-47d4-b8a6-c7886114568e.png)

Reply:
So that line should probably read `<signal device-variant="a" driver="smc" name="nrd" function="a"/>`.

Reply:
Can this be "simply" solved by adapting the from_identifier code to recognize the two-letter package types? Everything else should then work fine.